sportsref/utils.py

```diff
@@ -61,14 +61,14 @@ def parseTable(table):
         return pd.DataFrame()
```

Comment:
Again, these changes should instead be added by merging master into ncaaf (or by `git checkout master sportsref/utils.py` if necessary).